pkgs/development/python-modules/cadquery/default.nix

```diff
@@ -0,0 +1,88 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pyparsing
+, opencascade
+, stdenv
+, python
+, cmake
+, swig
+, ninja
+, smesh
+, freetype
+, libGL
+, libGLU
+, libX11
+, six
+}:
+
+let 
+  pythonocc-core-cadquery = stdenv.mkDerivation {
```

Comment:
Why is this not a package expose in `pythonPackages`?

Reply:
It's a branch of pythonocc modified by the CadQuery devs and used exclusively (as far as I know) for CadQuery.

Reply:
@adisbladis, knowing that, are you satisfied pythonocc-core-cadquery should not be exposed in pythonPackages? If so I'll mark this as resolved.